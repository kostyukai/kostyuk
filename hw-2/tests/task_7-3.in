Qjdksad12Qsd123Q!
