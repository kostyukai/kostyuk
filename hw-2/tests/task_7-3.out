QQjjddkkssaadd1122QQssdd112233QQ
